apps/style/netsim/style.scss

```diff
@@ -55,6 +55,55 @@ $wide-column-width: 121px;
   left: 0;
   right: 0;
 
+  // Alerts
+
+  .netsim-alert-container {
+    position: absolute;
+    top: -15px;
+    left: 50%;
+    width: 450px;
+    margin-left: -225px; // -(width/2)
+    z-index: 99999;
```

Comment:
It was, actually! It looks like that is the smallest of the "large" values, and that's exactly where I wanted the notifications to go; right below all the things that darken the screen behind em, but above everything else